src/main/java/dev/socketmods/socketnukes/explosion/types/CubicExplosionType.java

```diff
@@ -127,7 +128,7 @@ public ExplosionMetaPackage explode(Level worldIn, BlockPos source, int stage, E
 
                 if (doBlocksDrop) {
                     ObjectArrayList<Pair<ItemStack, BlockPos>> objectarraylist = new ObjectArrayList<>();
-                    Collections.shuffle(meta.affectedBlocks, worldIn.random);
+                    Collections.shuffle(meta.affectedBlocks, new Random());
```

Comment:
why use new Random here and not use the one from the world?
Same for VanillaExplosionType

Reply:
Could a wrapper be made to make them compatible? If not we should at least base the new Random's seed on the world seed. (Global random from the world see and use those as the seed for these)
I feel like being deterministic, based on the world seed would be useful.

Reply:
They are not compatible, and it is not possible to get the world seed.

RandomSource is a completely new system, it cannot be wrapped around Java's native Random, and it's seed is not visible even with reflection.

Reply:
Shouldn't we follow Vanilla here and migrate to `net.minecraft.Util#shuffle`?